src/handlers/RequestHandler.ts

```diff
@@ -25,6 +25,7 @@ export type DefaultRequestBody =
   | Record<string, any>
   | DefaultRequestMultipartBody
   | string
+  | number
```

Comment:
Number is also a valid input when composing a `Response` instance, so this change should be fine. 

Just note that this change affects accepted types for _all_ mocked responses (not necessarily those you set via `ctx.json`, but also via `ctx.text`, `ctx.body`, etc.). 

```js
let res = new Response(2)
await res.text() // '2'
```